app/api/request_api_utils.py

```diff
@@ -7,12 +7,12 @@
 from app.utils.decorator_utils import http_response_namedtuple_converter
 
 
-BASE_MS_API_URL = "http://127.0.0.1:4000"
+BASE_MS_API_URL = "http://MS:5000"
```

Comment:
check the port.

Reply:
Yes it works on docker because we are connecting to a different container. I'll add environment variables for this.